packages/documentation/styles/code.scss

```diff
@@ -0,0 +1,77 @@
+code {
+	color: #091142;
+	tab-size: 2;
+	background: #e6e6e6;
+}
+
+pre code {
+	font-weight: 500;
+	background: transparent;
+}
+
+/* stylelint-disable selector-class-pattern */
+.hljs {
```

Comment:
it's highlightjs classes anyways so we can't follow any stylelint fixes for them

Reply:
add to the disable comment